comics/main.go

```diff
@@ -37,30 +35,55 @@ func getComic(url string) (*Comic, error) {
 	req, err := http.NewRequest("GET", url, nil)
 
 	if err != nil {
-		log.Fatal(err)
 		return nil, err
 	}
 
 	resp, err := client.Do(req)
 	if err != nil {
-		log.Fatal(err)
 		return nil, err
 	}
 	defer resp.Body.Close()
 	var comic Comic
 	err = json.NewDecoder(resp.Body).Decode(&comic)
 	if err != nil {
-		log.Fatal(err)
 		return nil, err
 	}
 	return &comic, nil
 }
 
+func getLastFiveComics(comic *Comic, comicSlice *[]Comic) error {
```

Comment:
I would have gone with something like this: `func getComics(startNum, length int) ([]Comic, error) {`
1. You generate the slice inside the function and return it (no need for it to be a pointer).
2. Someone reading the original func for the first time would wonder why it has a `*Comic` as an argument. It's clear from the context in `main`, but you want functions to be as independent of usage context as possible.